bridges/BrutBridge.php

```diff
@@ -0,0 +1,142 @@
+<?php
+class BrutBridge extends BridgeAbstract {
+	const NAME = 'Brut Bridge';
+	const URI = 'https://www.brut.media';
+	const DESCRIPTION = 'Returns 5 newest videos by category and edition';
+	const MAINTAINER = 'VerifiedJoseph';
+	const PARAMETERS = array(array(
+			'category' => array(
+				'name' => 'Category',
+				'type' => 'list',
+				'values' => array(
+					'News' => 'news',
+					'International' => 'international',
+					'Economy' => 'economy',
+					'Science and Technology' => 'science-and-technology',
+					'Entertainment' => 'entertainment',
+					'Sports' => 'sport',
+					'Nature' => 'nature',
+				),
+				'defaultValue' => 'news',
+			),
+			'edition' => array(
+				'name' => ' Edition',
+				'type' => 'list',
+					'values' => array(
+						'United States' => 'us',
+						'United Kingdom' => 'uk',
+						'France' => 'fr',
+						'India' => 'in',
+						'Mexico' => 'mx',
+				),
+				'defaultValue' => 'us',
+			)
+		)
+	);
+
+	const CACHE_TIMEOUT = 1800; // 30 mins
+
+	private $videoId = '';
+	private $videoType = '';
+	private $videoImage = '';
+
+	public function collectData() {
+
+		$html = getSimpleHTMLDOM($this->getURI())
+			or returnServerError('Could not request: ' . $this->getURI());
+
+		$results = $html->find('div.results', 0);
+
+		foreach($results->find('li.col-6.col-sm-4.col-md-3.col-lg-2.px-2.pb-4') as $index => $li) {
+			$item = array();
+
+			if ($index > 5) {
+				break;
+			}
+
+			$videoPath = self::URI . $li->children(0)->href;
+
+			$videoPageHtml = getSimpleHTMLDOMCached($videoPath, 3600)
+				or returnServerError('Could not request: ' . $videoPath);
+
+			$this->videoImage = $videoPageHtml->find('meta[name="twitter:image"]', 0)->content;
+
+			$this->processTwitterImage();
+
+			$description = $videoPageHtml->find('div.description', 0);
+
+			$item['uri'] = $videoPath;
+			$item['title'] = $description->find('h1', 0)->plaintext;
+
+			if ($description->find('div.date', 0)->children(0)) {
+				$description->find('div.date', 0)->children(0)->outertext = '';
+			}
+
+			$item['content'] = $this->processContent(
+				$description
+			);
+
+			$item['timestamp'] = $this->processDate($description);
+			$item['enclosures'][] = $this->videoImage;
+
+			$this->items[] = $item;
```

Comment:
One more detail. To limit the number of items add a check right after adding the items instead of checking at the top of the loop. This will break the loop as soon as possible, skipping the next iteration entirely.

```suggestion
			$this->items[] = $item;
			if (count($this->items) >= 5)
				break;
```

It also doesn't rely on the array index, which could change in the future.